package.json

```diff
@@ -14,15 +14,15 @@
     "check-links:external": "yarn build && yarn broken-link-checker:external",
     "migrate": "yarn workspace @okta/migrate-from-jekyll migrate",
     "test": "yarn dev & yarn test:test-local && yarn stop-all",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
     "test:unit": "jest tests/unit",
-    "test-only": "yarn test:unit && yarn test:update-web-driver && protractor tests/selenium/conf.js && yarn stop",
+    "test-only": "yarn test:unit && yarn cy:run && yarn stop",
     "test:test-local": "yarn test:wait-on-local && yarn test-only || (yarn stop && exit 1)",
-    "test:update-web-driver": "node ./scripts/update-se-drivers.js",
-    "test:stop-web-driver": "yarn fkill -s $(pgrep -f 'webdriver-manager/selenium' || echo 99999999)",
     "test:wait-on-local": "wait-for-localhost 8080 --use-get",
     "markdown-lint": "node scripts/markdown-check",
     "posttest": "yarn stop",
-    "stop": "if-env CI=true && exit 0 || yarn test:stop-web-driver || echo 'done'",
+    "stop": "if-env CI=true && exit 0 || echo 'done'",
     "stop-all": "if-env CI=true && exit 0 || pkill -f 'yarn' || echo 'done'",
```

Comment:
hm I thought it doesn't relate to protractor

Reply:
I think the `stop` command doesn't do anything now, so we can remove. May need to update or remove `posttest`.

`stop-all` is probably still needed since it forcefully kills the dev server that is started for the tests.

Reply:
seems `posttest` is not using at all

Reply:
gonna to remove it